### ISSUE TITLE
Discrepancy in performance compared to paper

### ISSUE THREAD

**Issue by netw0rkf10w** [nyoki-mtl/pytorch-segmentation#22]
Hi,

First of all, thanks for the code! I was looking for a PyTorch port of DeepLabv3+ that can use the [official TensorFlow checkpoints](https://github.com/tensorflow/models/blob/master/research/deeplab/g3doc/model_zoo.md). Your repo is the _only_ one I could find! Great work!

There seems however to be some discrepancy compared to the TensorFlow version.
I tried the [`xception65_coco_voc_trainval`](http://download.tensorflow.org/models/deeplabv3_pascal_trainval_2018_01_04.tar.gz) checkpoint on the Pascal VOC _test set_ with flipping and multi-scale (`[0.5:0.25:1.75]`) inference, and obtained **mIoU = 85.93630** (by uploading to the evaluation server), which far inferior to the reported **87.80%**.

In addition to the [`difference in BatchNorm's eps`](https://github.com/nyoki-mtl/pytorch-segmentation/issues/10) that you discovered previously, there should be some other details that affect the performance. Unfortunately I was unable to find them. I hope you or somebody can.

**Comment by JoOkuma** [comment]
@netw0rkf10w were you able to improve the performance to get closer to the TF version?
